q2_demux/_summarize/_visualizer.py

```diff
@@ -108,6 +123,7 @@ def summarize(output_dir: str, data: _PlotQualView, n: int=10000) -> None:
         count = 0
         for seq in _read_fastq_seqs(file):
             count += 1
+
```

Comment:
Edit: that was because I used a prior version of `flake8` which wanted two empty lines after a function or class declaration.  I'll fix my dev environment accordingly.  Thanks!